tinygrad/mlops.py

```diff
@@ -41,6 +41,14 @@ def forward(self, x:LazyBuffer) -> LazyBuffer:
 
   def backward(self, grad_output:LazyBuffer) -> LazyBuffer:
     return grad_output.binary_op(BinaryOps.DIV, self.x)
+  
+class Log2(Function):
```

Comment:
Why? Just change of base the log.

Reply:
my though was Mlop Log uses Unary op Log2, so it would be like calculating Log2 --> converting to Log --> converting to Log2 again, accumulating numerical error

Reply:
i'm getting the same results numerically with log2 base conversion from Log mlop, removing Log2 mlop